kafka-inmemory/resources/openshift-template.yaml

```diff
@@ -49,6 +49,16 @@ parameters:
   displayName: Kafka healthcheck timeout
   name: KAFKA_HEALTHCHECK_TIMEOUT
   value: "5"
+- description: Default replication factor for newly created topics
+  displayName: Default replication factor
+  name: KAFKA_DEFAULT_REPLICATION_FACTOR
+  value: "1"
+- description: |
+             Replication factor for offsets (offsets.topic.replication.factor) 
+             and transactions (transaction.state.log.replication.factor)
```

Comment:
Why specifying the configuration parameter name and not "offsets topics" and "transactions topics" ? You used something like that before saying "newly created topics".